src/java/org/apache/cassandra/cache/ChunkCache.java

```diff
@@ -62,6 +67,25 @@ public class ChunkCache
     private final static Logger logger = LoggerFactory.getLogger(ChunkCache.class);
 
     public static final int RESERVED_POOL_SPACE_IN_MB = 32;
+    private static final int INITIAL_CAPACITY = Integer.getInteger("cassandra.chunkcache_initialcapacity", 16);
+    private static final boolean ASYNC_CLEANUP = Boolean.parseBoolean(System.getProperty("cassandra.chunkcache.async_cleanup", "true"));
```

Comment:
`Boolean.parseBoolean(System.getProperty....`
what's the best way to do this in Converged Cassandra ?

Reply:
You can find several places where we do it this exact way. However, I think that `CassandraRelevantProperties` might be the preferred way.

Reply:
In C* it must go through `CassandraRelevantProperties`; CC should adopt this as well.

Actually, could we not use the number of threads below to control this? (0 threads -> sync cleanup; >=1 -> async)